.gitignore

```diff
@@ -14,4 +14,4 @@ hs_err_pid*
 .idea/
 *.iml
 .gradle/
-
+build/
```

Comment:
I can do that in a day or so when I get back to my dev machine - or you could revert it after merge if you don't want to wait.
I'm interested in knowing why you would want to remove it though? I compiled the project using IDEA with the gradle plugin and this was the default directory where it put the build artifacts. Given that we ignore the .idea directory it would seem to make sense to also ignore the build directory. Without the ignore git status shows all the class files and makes it difficult to see if any real files have been added. Given that there is unlikely to be a source directory called build, what's the downside of adding it to the ignore file?